src/colmap/scene/database_cache.h

```diff
@@ -98,7 +98,7 @@ class DatabaseCache {
   const class Image* FindImageWithName(const std::string& name) const;
 
   // Setup PosePriors for PosePriorBundleAdjustment
-  bool SetupPosePriors();
+  bool SetupPosePriors(GPSTransform::CartesianFrame cartesian_frame_from_lla);
```

Comment:
nit: using our current naming conventions `cartesian_frame_from_lla` implies a transformation. I suggest renaming this to just `cartesian_frame`.